# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderD3D11TextureHost.cpp
# Commit: 71b6060b6015
# Full Hash: 71b6060b60151e157e94110d28fb51d76bb6415a
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2023-12-08 04:18:28
# Regressor Bug: 1863914
# File Overlap Count: 1
# Description:
#   Bug 1863914: Use multiple shmem buffers for remote canvas recording. r=aosmond
#   
#   This replaces the use of a single large ring buffer.
#   The buffers are still processed in parallel and are recycled to reduce
#   allocation. Events that do not fit in the default sized buffer have a separate
# ==============================================================================

diff -r bf28b3c066e6 -r 71b6060b6015 gfx/webrender_bindings/RenderD3D11TextureHost.cpp
--- a/gfx/webrender_bindings/RenderD3D11TextureHost.cpp	Thu Dec 07 17:10:06 2023 +0000
+++ b/gfx/webrender_bindings/RenderD3D11TextureHost.cpp	Thu Dec 07 17:27:24 2023 +0000
@@ -13,6 +13,7 @@
 #include "RenderCompositorD3D11SWGL.h"
 #include "ScopedGLHelpers.h"
 #include "mozilla/DebugOnly.h"
+#include "mozilla/gfx/CanvasManagerParent.h"
 #include "mozilla/gfx/Logging.h"
 #include "mozilla/layers/GpuProcessD3D11TextureMap.h"
 #include "mozilla/layers/TextureD3D11.h"